TestClasses/BaseTest.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import os
+
+class BaseTest():
+    def __init__(self):
+        self.action = None
+        self.sourcePath = None
+        self.destinPath = None
+        self.args = None
+        self.cmds = {'root://':'xrd', 'https://':'gfal', 'davs://':'gfal'}       
+        self.sitesList = {'CEPH-SVC16':['ceph-svc16.gridpp.rl.ac.uk', 'dteam:/test/']} #'LHCONE':['eoslhcb.cern.ch', 'dteam:/test/'], 'LHCOPN':['ce01-lhcb-t2.cr.cnaf.infn.it'], 'DCACHE':'hostname3'}
+        self.redirect = {'INT_MANAGER':['xrootd.echo.stfc.ac.uk', 'dteam:/test/']} 
```

Comment:
INT_MANAGER would be echo-internal-manager01.gridpp.rl.ac.uk, xrootd.echo.stfc.ac.uk would be something like PROD_MANAGER